sources/web/datalab/static.ts

```diff
@@ -159,14 +160,51 @@ function sendUserCustomTheme(userId: string, response: http.ServerResponse): voi
 }
 
 /**
+ * Returns true if this path should return an experimental UI resource
+ * @param path the incoming request path
+ */
+export function isExperimentalResource(pathname: string) {
+  const experimentalUiEnabled = process.env.DATALAB_EXPERIMENTAL_UI;
```

Comment:
Is this the only place that DATALAB_EXPERIMENTAL_UI will be referenced in the ts code? If there might be other places where you have to make this check, I think I'd rather see a method experimentalUiIsEnabled() that returns true if that env var is set, and have all checks go through that.